Sources/Private/Utility/Primitives/VectorsExtensions.swift

```diff
@@ -54,7 +54,7 @@ extension Double {
 /**
  Needed for decoding json {x: y:} to a CGPoint
  */
-struct Vector2D: Codable {
+public struct Vector2D: Codable, Hashable {
```

Comment:
Same question here re: moving this out of the private folder now that it's public API?